skrub/_reporting/_data/templates/base.css

```diff
@@ -0,0 +1,452 @@
+:host {
```

Comment:
nothing comes from an external source except the files in `pure-3.0.0` (a css reset & basic styling for tables). I'll add more comments and see if it can be reduced/reorganized a bit